canonical/ubuntu.com
# /telco content refresh

## Done

- Applied content refresh as per [doc](https://docs.google.com/document/d/1a3otoXCbXtddJG2HCnYIIHJiDxvG-pzFPtLobVB4-vw/edit)

## QA

- View the site locally in your web browser at: https://ubuntu-com-13395.demos.haus//telco
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check against doc

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-6454
